### PR TITLE
Create evalopensourcevspropietary.md

### PR DESCRIPTION
**Pull request by scoker-me**

### CODE REVIEW

File: learning-content/beginners-guides/evalopensourcevspropietary.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
+# Evaluating Open Source versus Proprietary Software for Data Workflows in the Social Sector: A Beginner’s Guide
+### Stephanie Coker and Benjamin Kinsella
+
+##### Introduction:
+Given the central role digital technologies play across many industries, the social sector has benefited enormously from Open Source software (OSS). In 2020, GitHub, the world’s largest internet hosting for software development, observed over 1.9 billion contributors to projects, representing a 25% increase in open source repositories from 2019. The proliferation of OSS offers social sector leaders and practitioners the opportunity to innovate and collaborate using digital technologies, providing novel alternatives to proprietary software.
```
Comment (malakumar85) [inline learning-content/beginners-guides/evalopensourcevspropietary.md:5]: Can you rephrase this opening paragraph using the language on the github.com homepage - "the largest software development platform in the world" - or similar could work.  Also not that it's 1.9 **contributions,** not individual contributors.

File: learning-content/beginners-guides/evalopensourcevspropietary.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+##### Key terminology 
+In this Beginner’s Guide, OSS is broadly defined as software that is freely distributed and uploaded to an accessible repository that provides source code. More specifically, code, data, or content has to be under an open source license, insofar that the code base is accessible at no cost (see The Open Source Initiative for further outline and definitions). Proprietary software, on the other hand, is copyrighted, such that limits against use, distribution, and/or modification are imposed by the developer. Moreover, along the continuum of software development licenses also exists a business model known as Open Core. That is, Open Core is generally understood as the monetization of commercially produced OSS, proprietary add-ons that provide additional features that come with a commercial license. Examples of Open Core Software include many Service (SaaS) products, like Kafka, a data streaming service under Apache 2.0, Oracle’s MySQL database, Neo4j with the enterprise addition under a commercial license, among many others. To conceptualize the range of differences between OSS, Open Core, and proprietary software, see Figure 1:
```
Comment (malakumar85) [inline learning-content/beginners-guides/evalopensourcevspropietary.md:14]: For the line: "Proprietary software, on the other hand, is copyrighted, such that limits against use, distribution, and/or modification are imposed by the developer."

should it be "imposed by the owner"?

**Review by malakumar85 — COMMENTED** [review]
@scoker-me @bjk127 - I added a few changes to make. Looks great overall, congrats! Almost ready to publish.

**Comment by malakumar85** [comment]
@scoker-me @bjk127 - following up on this. I can merge the PR after the edits have been made, thanks!

**Comment by malakumar85** [comment]
@scoker-me @bjk127 - this PR has been open for over a month. Can we get the final reviews to push it through?

**Comment by bjk127** [comment]
@malakumar85 Thanks for following up on this! I must have missed these notifications. I'll get on this right now.

File: learning-content/beginners-guides/evalopensourcevspropietary.md (hunk 1 above)
Comment (bjk127) [reply learning-content/beginners-guides/evalopensourcevspropietary.md]: Done. Thanks!

File: learning-content/beginners-guides/evalopensourcevspropietary.md (hunk 2 above)
Comment (bjk127) [reply learning-content/beginners-guides/evalopensourcevspropietary.md]: Revised.

**Comment by bjk127** [comment]
Hi @malakumar85 , I revised the edits you suggested. Please let me know if the submitted changes didn't go through.

**Comment by malakumar85** [comment]
Looks good, @bjk127 ! Can you add your review? We can merge the pull request after.

**Comment by bjk127** [comment]
@malakumar85 pressed "Add Review" so should be up now! Apologies for the bad commit skills!
